packages/sdk/explorer/src/sourcify.ts

```diff
@@ -21,6 +21,9 @@ const PROXY_IMPLEMENTATION_GETTERS = [
   'implementation',
 ]
 
+const PROXY_IMPLEMENTATION_POSITION_UUPS =
```

Comment:
what is UUPS and why do we want a hex hardcoded into the code?

Reply:
UUPS is [Proxy pattern](https://docs.openzeppelin.com/contracts/4.x/api/proxy#transparent-vs-uups). Since this value is also hardcoded in [OpenZeppelin smart](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8b4b7b8d041c62a84e2c23d7f6e1f0d9e0fc1f20/contracts/proxy/ERC1967/ERC1967Utils.sol#L35) contract there is no other way than to copy&paste it

Reply:
Comment&link added

Reply:
thanks!